### PR TITLE
fix(css): improve color-mixer tool

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- addresses https://github.com/mdn/content/pull/34371#discussion_r1661199256

The PR
- makes color pickers keyboard navigable
- fixed width for labels so UI layout won't change

### CODE REVIEW

File: tools/color-mixer/index.html [hunk 1]
```diff
@@ -123,13 +140,25 @@
...
-      <div id="color-one" class="color-one"></div>
+      <label>color-two</label>
+
+      <button
+        id="color-one"
+        type="button"
+        class="color-one"
+        role="button"
+        aria-label="toggle color picker dialog"
+        tabindex="0"></button>
```
Comment (pepelsbey) [inline tools/color-mixer/index.html:153]:
```suggestion
      <button
        id="color-one"
        type="button"
        class="color-one"
        aria-label="toggle color picker dialog"
      ></button>
```

File: tools/color-mixer/index.html [hunk 2]
```diff
@@ -123,13 +140,25 @@
...
       <div id="mixed-color"></div>
-      <div id="color-two" class="color-two"></div>
+      <button
+        id="color-two"
+        type="button"
+        class="color-two"
+        role="button"
+        aria-label="toggle color picker dialog"
+        tabindex="0"></button>
```
Comment (pepelsbey) [inline tools/color-mixer/index.html:161]:
```suggestion
      <button
        id="color-two"
        type="button"
        class="color-two"
        aria-label="toggle color picker dialog"
      ></button>
```

**Review by pepelsbey — COMMENTED** [review]
This looks great! Thank you for improving the accessibility of this tool :)

I’d also suggest removing the `role` and `tabindex` attributes from buttons since they already have this role and are focusable by default. There’s one selector in the script that relies on the `tabindex` attribute, but it likely from the picker. I haven’t noticed any difference in the behavior.

Also, I noticed an error in DevTools because the source map file mentioned in the JS file is missing:

```
//# sourceMappingURL=pickr.es5.min.js.map
```

How about we remove this line to calm it down a bit? :)

**Comment by OnkarRuikar** [comment]
>  There’s one selector in the script that relies on the tabindex attribute, but it likely from the picker.

Yes, it's from picker. And the current changes don't affect it.

**Review by pepelsbey — APPROVED** [review]
Perfect! Thank you :)

**Pull request merged**